src/util/catchSetupAndConfig.js

```diff
@@ -32,18 +22,24 @@ const commands = [
   'help',
   'version',
 ];
-const commandIsNotValid = command => !commands.includes(command);
+const commandIsNotValid = (command) => !commands.includes(command);
 
 module.exports = async function catchSetupAndConfig(path, command) {
-  if (commandIsNotValid(command) || ['help', 'version', 'create'].includes(command)) return true;
-
   const awsConfigExistsWithRegionSet = getRegion();
   if (!awsConfigExistsWithRegionSet) {
-    bamWarn('AWS credentials have not been set up.  For instructions, please visit:');
+    bamWarn(
+      'AWS credentials have not been set up.  For instructions, please visit:',
+    );
     log('https://docs.aws.amazon.com/cli/latest/topic/config-vars.html');
     return false;
   }
 
+  if (
```

Comment:
Here the guard clause is moved down because we would like to ensure the AWS credentials are setup no matter what, correct?